app/livechat/client/views/app/tabbar/visitorEdit.js

```diff
@@ -106,9 +155,17 @@ Template.visitorEdit.events({
 		userData.name = event.currentTarget.elements.name.value;
 		userData.email = event.currentTarget.elements.email.value;
 		userData.phone = event.currentTarget.elements.phone.value;
+		userData.livechatData = {};
```

Comment:
I have a suggestion about the process of saving custom fields in both `Visitor` or `Department` scope:
What do you think of adding a new field(optional) on the `Custom Field` configuration, allowing to define a `regex validation`?
Then, you just need to check if a `custom field` has a validation(regex) or not(If so, run the validation(s) before saving..)

Reply:
Ok! I will add

Reply:
Hey @renatobecker! I added a regex validation for custom fields and rebased the branch

Reply:
> Hey @renatobecker! I added a regex validation for custom fields and rebased the branch

Awesome! I'm gonna review this PR right now.
Thank you in advance.